app/src/main/res/values/strings.xml

```diff
@@ -66,4 +66,7 @@
     <string name="scanning">Scanning</string>
     <string name="skipping_folder">Skipping %1$s</string>
     <string name="enough">Enough</string>
+    <string name="select">Select</string>
```

Comment:
Maybe better to call it "Navigate"? Because "select" sounds like it's possible to select more than one tag. Also "Remove" is probably better to rename into "Untag", because no data is really deleted.